client/src/Components/LandingPage.js

```diff
@@ -3,22 +3,23 @@ import PropTypes from "prop-types";
 import Footer from "./Footer.js";
 import HeroBanner from "./HeroBanner.js";
 import Grid from "@material-ui/core/Grid";
-import TopicTiles from "../TopicTiles";
-// import TopicTiles from './TopicTiles.js';
+import TopicTiles from "../topicTiles";
 
 const LandingPage = props => {
-  console.log(props);
   return (
     <Grid container>
       {/* <Header /> */}
       <Grid item sm={12}>
         <HeroBanner name={"Octavius"} />
       </Grid>
-      <TopicTiles
-        allTopics={props.allTopics}
-        handleTopicTileClick={props.handleTopicTileClick}
-        favoritedTopics={props.favoritedTopics}
-      />
+      <Grid style={{ paddingBottom: 5 }}>
+        <TopicTiles
+          allTopics={props.allTopics}
+          handleTopicTileClick={props.handleTopicTileClick}
+          favorites={props.favorites}
+        />
+        <HeroBanner name={"Octavius"} />
```

Comment:
Why did the hero banner get added here as well? it should have already been brought in.